core/operations/document_lookup_in_all_replicas.hxx

```diff
@@ -141,59 +150,64 @@ struct lookup_in_all_replicas_request {
             };
 
             auto ctx = std::make_shared<replica_context>(std::move(h), nodes.size());
+            auto access_deleted = false;
+            if (enable_access_deleted) {
+              access_deleted = config->capabilities.supports_subdoc_access_deleted();
+            }
```

Comment:
In my opinion, it would be better for the core replica lookup_in requests to simply forward the value for `access_deleted` that the caller set, and let the server decide if it's valid or not. In that case, users of the core API can decide what to do when the feature isn't supported. We can implement the cap check in the transactions library instead.

Reply:
Good point, have made changes accordingly 👍